### PR TITLE
Payment counting

### LINKED ISSUE #43: Fix payment counting

### PR DESCRIPTION
**Pull request by ref-humbold**
Fix #43

### CODE REVIEW

File: app/src/conferences/models.py [hunk 1]
```diff
@@ -235,18 +235,18 @@ class Meta:
     bus = models.ForeignKey(Bus, null=True, blank=True, on_delete=models.SET_NULL)
 
     # Day 1 (Coming)
-    accomodation_day_1 = models.BooleanField(default=False)
     dinner_1 = models.BooleanField(default=False)
```
Comment (Bozydarek) [inline app/src/conferences/models.py]: I think that we should change name of this field to `dinner_day_1` and the remaining analogously

File: app/src/conferences/forms.py [hunk 2]
```diff
@@ -60,9 +53,6 @@ class Meta(UserPreferencesWithOrgForm.Meta):
         model = UserPreferences
         exclude = ['user', 'zosia', 'payment_accepted', 'bonus_minutes']
 
-    # Yes, it's required by default. But it's insane - better be verbose than misunderstood.
-    accepted = forms.BooleanField(required=True)
```
Comment (Bozydarek) [inline app/src/conferences/forms.py]: So it is True by default now?

File: app/src/conferences/models.py [hunk 3]
```diff
@@ -286,41 +286,34 @@ class Meta:
...
-            'breakfast_2': breakfast_price,
-            'accomodation_day_2': self.zosia.price_accomodation,
-            'dinner_2': dinner_price,
-            'breakfast_3': breakfast_price,
-            'accomodation_day_3': self.zosia.price_accomodation,
-            'dinner_3': dinner_price,
-            'breakfast_4': breakfast_price
-        }[option_name]
+    def _price_for(self, chosen):
+        if not chosen["accomodation_day"] and not chosen["dinner"] and not chosen["breakfast"]:
```
Comment (Bozydarek) [inline app/src/conferences/models.py]: If we will add `_day` to both `dinner` and `breakfast` you will be able to remove it from here:
`accomodation_day`->`accomodation`

File: app/src/conferences/models.py [hunk 4]
```diff
@@ -286,41 +286,34 @@ class Meta:
...
-        for group in payment_groups:
-            if all(map(lambda d: self._pays_for(d), group)):
-                payment += self.zosia.price_whole_day
-            else:
-                for g in group:
-                    if self._pays_for(g):
-                        payment += self._price_for(g)
+        for accommodation, meals in PAYMENT_GROUPS.items():
+            chosen = {
+                accommodation[:-2]: self._pays_for(accommodation),
```
Comment (Bozydarek) [inline app/src/conferences/models.py]: I have to read whole PR to understand why you are removing last 2 characters, please add some comment about it

File: app/src/conferences/forms.py (hunk 2 above)
Comment (ref-humbold) [reply app/src/conferences/forms.py]: This field wasn't even written to model! I suppose it should've been used to accept terms and conditions here: https://github.com/ksiuwr/zosia16-site/blob/f4bb31ecef26bd73f9a3e7919011cd7d50c6ec79/app/src/conferences/templates/conferences/register.html#L27

File: app/src/conferences/models.py (hunk 1 above)
Comment (ref-humbold) [reply app/src/conferences/models.py]: Or remove it from `accommodation_day_x`.

File: app/src/conferences/forms.py (hunk 2 above)
Comment (Bozydarek) [reply app/src/conferences/forms.py]: Now, on second thought, I think that we should restore this and put here accepting of the ZOSIA T&C and at the same time remove it from the first registration form. What do you think about this?

Comment (ref-humbold) [reply app/src/conferences/forms.py]: Done

**Pull request merged**